ember-cli/2017-08/august-17.md

```diff
@@ -0,0 +1,7 @@
+# ember-cli Meeting Notes - 2017-08-17
+
+## Attendees
+
+## Agenda
```

Comment:
Discuss and review with the team the result of @kratiahuja and @rwjblue's conversation on fastboot + engines + meta config.